retrievalmarket/impl/client.go

```diff
@@ -110,15 +84,15 @@ func (c *client) Retrieve(ctx context.Context, pieceCID []byte, params retrieval
 
 	go func() {
 		evt := retrievalmarket.ClientEventError
-		converted, err := cid.Cast(pieceCID)
+		//converted, err := cid.Cast(pieceCID)
```

Comment:
Is there a reason to keep this commented-out code or did you mean to delete it?

Reply:
Please delete